internal/setup/setup.go

```diff
@@ -553,6 +562,12 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 		if len(details.Components) == 0 {
 			return nil, fmt.Errorf("%s: archive %q missing components field", fileName, archiveName)
 		}
+		switch details.Pro {
+		case "", ProApps, ProFIPS, ProFIPSUpdates, ProInfra:
+		default:
+			logf("Ignoring archive %q (invalid pro value: %q)...", archiveName, details.Pro)
```

Comment:
See earlier note about the style of log messages.

Reply:
Done in https://github.com/canonical/chisel/pull/167/commits/3adec2115f52d7c843696754c451811025656d48.